proto/api_v3/query_service.proto

```diff
@@ -48,13 +48,13 @@ message TraceQueryParameters {
   // Attributes are matched against Span and Resource attributes.
   // At least one span in a trace must match all specified attributes.
   map<string, string> attributes = 3;
-  // Span min start time.
+  // Span min start time in. RFC-3339ns format. Required.
   google.protobuf.Timestamp start_time_min = 4;
-  // Span max start time.
+  // Span max start time. RFC-3339ns format. Required.
   google.protobuf.Timestamp start_time_max = 5;
-  // Span min duration.
+  // Span min duration. Golang duration format. Required.
```

Comment:
Duration was not a required param previously.

> Golang duration format.

Not sure what it means in proto definition, the format is defined by google.protobuf.Duration, not by Golang.

Reply:
for the REST API it is golang's time format. Can you tell that by looking at https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration?

Reply:
that's probably the grpc-gateway's interpretation? In that link many types have a dedicated JSON section, but Duration does not, and since it's just a composite proto type I think its default JSON representation would be a struct, not a Go duration string, so it must be somehow handled differently in the translation. The same probably applies to the Timestamp type.

Reply:
I don't know, but suing go time strings work. Hence I have documentedthat.